src/App.js

```diff
@@ -1,25 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
```

Comment:
## Move the app component
- [ ] The `App.js` component needs to be moved to the `src/components` directory.